.gitignore

```diff
@@ -239,9 +239,6 @@ Network Trash Folder
 Temporary Items
 .apdisk
 
-#dotnet.exe output
-tools/
```

Comment:
Strange. Why **hasn't** this .gitignore affected past updates to the files under `tools/`?